### PR TITLE
Update AWS page

### PR DESCRIPTION
**Pull request by awjuliani**

### CODE REVIEW

File: docs/Training-on-Amazon-Web-Service.md [hunk 1]
```diff
@@ -1,53 +1,59 @@
...
-3. Run the following commands to install Xorg:
+
+## Testing
+
+If all steps worked correctly, upload a built Linux environment executable (with `headless` mode selected during build-time) to the instance, and test it from Python using:
+
+```
+python3
+```
+```
```
Comment (mmattar) [inline docs/Training-on-Amazon-Web-Service.md]: add python tag for markdown

File: docs/Training-on-Amazon-Web-Service.md [hunk 2]
```diff
@@ -1,53 +1,59 @@
 # Training on Amazon Web Service
 
-This page contains instructions for setting up an EC2 instance on Amazon Web Service for use in training ML-Agents environments. Current limitations of the Unity Engine require that a screen be available to render to. In order to make this possible when training on a remote server, a virtual screen is required. We can do this by installing Xorg and creating a virtual screen. Once installed and created, we can display the Unity environment in the virtual environment, and train as we would on a local machine. 
+This page contains instructions for setting up an EC2 instance on Amazon Web Service for use in training ML-Agents environments. You can run "headless" training if none of the agents in the environment use visual observations.
```
Comment (mmattar) [inline docs/Training-on-Amazon-Web-Service.md]: "use in training ML-Agents environments" --> "training in ML-Agents"

File: docs/Training-on-Amazon-Web-Service.md [hunk 3]
```diff
@@ -1,53 +1,59 @@
...
+2. Move `python` sub-folder of this repo to remote instance, and set as working directory.
 2. Install the required packages with `pip3 install .`.
-3. Run the following commands to install Xorg:
+
+## Testing
+
+If all steps worked correctly, upload a built Linux environment executable (with `headless` mode selected during build-time) to the instance, and test it from Python using:
+
+```
+python3
```
Comment (mmattar) [inline docs/Training-on-Amazon-Web-Service.md]: Not sure we need this.

Comment (awjuliani) [reply docs/Training-on-Amazon-Web-Service.md]: Which piece isn't needed? The testing section?

Comment (mmattar) [reply docs/Training-on-Amazon-Web-Service.md]: Just lines 24 to 26.

File: docs/Training-on-Amazon-Web-Service.md [hunk 4]
```diff
@@ -1,53 +1,59 @@
...
 
 1. To begin with, you will need an EC2 instance which contains the latest Nvidia drivers, CUDA8, and cuDNN.  There are a number of external tutorials which describe this, such as:
     * [Getting CUDA 8 to Work With openAI Gym on AWS and Compiling TensorFlow for CUDA 8 Compatibility](https://davidsanwald.github.io/2016/11/13/building-tensorflow-with-gpu-support.html)
     * [Installing TensorFlow on an AWS EC2 P2 GPU Instance](http://expressionflow.com/2016/10/09/installing-tensorflow-on-an-aws-ec2-p2-gpu-instance/)
     * [Updating Nvidia CUDA to 8.0.x in Ubuntu 16.04 – EC2 Gx instance](https://aichamp.wordpress.com/2016/11/09/updating-nvidia-cuda-to-8-0-x-in-ubuntu-16-04-ec2-gx-instance/)
-2. Move `python` to remote instance.
+
+## Installing ML-Agents
+
+2. Move `python` sub-folder of this repo to remote instance, and set as working directory.
```
Comment (jo3w4rd) [inline docs/Training-on-Amazon-Web-Service.md]: Move the `python` sub-folder of this repo to the remote instance and set as working directory.

(Two "the's" and remove comma.)

File: docs/Training-on-Amazon-Web-Service.md [hunk 5]
```diff
@@ -1,53 +1,59 @@
...
 2. Install the required packages with `pip3 install .`.
-3. Run the following commands to install Xorg:
+
+## Testing
+
+If all steps worked correctly, upload a built Linux environment executable (with `headless` mode selected during build-time) to the instance, and test it from Python using:
```
Comment (jo3w4rd) [inline docs/Training-on-Amazon-Web-Service.md]: To verify that all steps worked correctly:

1. In the Unity Editor, load a project containing an ML-Agents environment (you can use one of the example environments if you have not created your own).
2. Open the **Build Settings** window (menu: **File** > **Build Settings**).  
3. Select **Linux** as the **Target Platform**.

[Editors note: does Architecture matter?]

4. Check **Headless Mode** (unless you have enabled a virtual screen following the instructions below).
5. Click **Build** to build the Unity environment executable.
6. Upload the executable to your EC2 instance.
7. Test the instance setup from Python using:

**Pull request merged**